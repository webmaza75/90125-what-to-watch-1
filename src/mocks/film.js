const film = {
  genre: [`Drama`, `Thriller`],
  title: `Aviator`,
  desc: ``,
  picture: `aviator.jpg`,
  year: 2016,
  src: ``
};

export default film;
