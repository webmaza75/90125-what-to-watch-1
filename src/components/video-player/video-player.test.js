import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';
import film from '../../mocks/film.js';

window.HTMLMediaElement.prototype.play = () => {};

const {src} = film;

it(`VideoPlayer correctly renders`, () => {
  const videoPlayer = renderer
    .create(<VideoPlayer
      src={src}
      isPlaying={true}
    />)
    .toJSON();

  expect(videoPlayer).toMatchSnapshot();
});
