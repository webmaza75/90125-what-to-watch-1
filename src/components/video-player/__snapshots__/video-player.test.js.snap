// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer correctly renders 1`] = `
<video
  className="player__video"
  muted={true}
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
/>
`;
